README.md

```diff
@@ -41,39 +41,49 @@ This is the dependency you need:
 </dependency>
 ```
  
-Then, you implement `com.yegor256.rpm.Storage` interface.
+Then, use one of the existing `com.artipie.asto.Storage` implementations to create the `Storage`,
+[`FileStorage`](https://github.com/artipie/asto/blob/master/src/main/java/com/artipie/asto/fs/FileStorage.java), 
+[`S3`](https://github.com/artipie/asto/blob/master/src/main/java/com/artipie/asto/s3/S3Storage.java) 
+and other storages are supported. Or you can implement `com.artipie.asto.Storage` by yourself.
 
 Then, you make an instance of `Rpm` class with your storage
-as an argument. Finally, you put your artifacts to the storage
-and instruct `Rpm` to update the meta info:
+as an argument. Finally, you put your artifacts to the storage specifying repository key 
+(`rpm-repo` in our example) and instruct `Rpm` to update the meta info:
 
 ```java
 import com.artipie.rpm.Rpm;
-Rpm rpm = new Rpm(storage);
-rpm.update("test/my.rpm");
+final Rpm rpm = new Rpm(storage);
+rpm.batchUpdate(new Key.From("rpm-repo"));
 ```
 
-To update the whole repository with all RPM files use `batchUpdate` method of `Rpm` class:
-```java
-import com.artipie.rpm.Rpm;
-Rpm rpm = new Rpm(storage);
-rpm.batchUpdate("test");
-```
-
-Read the [Javadoc](http://www.javadoc.io/doc/com.yegor256/rpm-files)
+Read the [Javadoc](https://www.javadoc.io/doc/com.artipie/rpm-adapter/latest/index.html)
 for more technical details.
 
-### Naming policy
+### Naming policy and checksum computation algorithm
 
 RPM may use different names to store metadata files in the package,
-by default `NamingPolicy.DEFAULT` will be used, to change naming policy use
-secondary constructor of `Rpm` to configure it. For instance to add `SHA1` prefixes for metadata files
-use `new Rpm(storage, new NamingPolicy.Sha1Prefixed())`.
+by default we use `StandardNamingPolicy.PLAIN`, to change naming policy use
```

Comment:
@olenagerasimova This should be the end of sentence like `by default we use StandardNamingPolicy.PLAIN. To change naming policy use...`